### PR TITLE
Styling the home page

### PR DESCRIPTION
**Pull request by MagdaSlifierz**
## Description

When a user visits the home page without being logged into an existing list, the navigation bar links 'Home,' 'Add Item,' and 'List' are concealed. Instead, they encounter a welcoming message and the Smart Shopping List logo. Additionally, the user is presented with two styled buttons, allowing them to choose between creating a new list or joining an existing one based on their preference. When the user types the name of an existing list, then the Alert is displayed.

## Note

Please note that the header style has already been updated in the open PR #35. 
As a team, we need to agree on whether we want to take the following actions for the Home Page: hide the links, keep them inactive, or completely hide the new navbar."

## Related Issue

closes#16

## Acceptance Criteria

- [x] The header should be hidden if the user is not logged into a list
- [x] The Home page should be styled  

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
| ✓ | Add styling              |

### Before
Screenshot 2023-10-03 at 7 45 25 PM

## After
Screenshot 2023-10-03 at 7 45 41 PM

Screenshot 2023-10-03 at 9 01 29 PM

## Testing Steps / QA Criteria
1. Go to the main branch in your code and get all the latest changes by using the command `git pull`.
2.  Switch to the `home-page-styling` branch.
3. Start your development environment by running the command `npm start`.
4. Access your browser's developer tools, and go to the Local Storage section under the Application tab. Clear the token stored there.
5. Refresh your website, and you should now observe the home page.

### CODE REVIEW

**Review by codecaviette — APPROVED** [review]
Looks great! On smaller viewports, I noticed the "Say hello..." text is left-aligned, and I think it makes more sense to be center-aligned. Once that change is made, I approve!

**Comment by MagdaSlifierz** [comment]
Thanks for catching that Jenny. I just pushed the changes.

**Review by lyleschemmerling — APPROVED** [review]

**Comment by lyleschemmerling** [comment]
When I follow the link to the hosted branch for this PR on github I see a broken image link

**Comment by avinashi10** [comment]
This UI requires users to scroll a lot to see all their options - a more common UI for 'create new' vs 'join existing' (similar to 'sign up' vs 'log in') might be something like the following:
image
image

Some suggestions (I'm happy to work on these together):
-  removing the logo from the page (since it will already be in the header)
- pulling the updated header from main and resolving conflicts so that the new header design is visible but your logic for hiding the navigation tabs and only showing the app title remains
- updating the background color to something very light/off-white
- creating a box  in the center of the page (maybe with a drop shadow effect) similar to the images above with two tabs that read 'create new list' and 'join existing list'

File: src/views/Layout.jsx [hunk 1]
```diff
@@ -10,35 +10,39 @@ import {
...
 export function Layout() {
+	const location = useLocation();
+	const isHomePage = location.pathname === '/';
+
 	return (
 		<Flex as="div" direction="column" h="100vh">
 			<Center as="header" bg="brand.off_white" p={5} boxShadow="md">
-				<Heading as="h1" size="xl" color="brand.navy">
+				<Heading as="h1" size={isHomePage ? '4xl' : 'xl'} color="brand.navy">
 					PredictaCart
```
Comment (avinashi10) [inline src/views/Layout.jsx]: This logic will need to be merged with the updated header in the main branch

**Pull request merged**